sdk/core/allocator/main.cc

```diff
@@ -863,22 +863,53 @@ __cheriot_minimum_stack(0x90) ssize_t
 	return cap->quota;
 }
 
-__cheriot_minimum_stack(0xd0) int heap_quarantine_empty()
+__cheriot_minimum_stack(0xd0) int heap_quarantine_flush(Timeout *timeout)
 {
 	STACK_CHECK(0xd0);
```

Comment:
Aside: now that we have `__cheriot_minimum_stack__` can we introduce a variant of this macro that doesn't need repeat the stack size?